app/services/populi_service.rb

```diff
@@ -41,4 +42,14 @@ def update_student_attendance(instance_id, meeting_id, person_id, status)
   def course_meetings(course_id)
     PopuliAPI.get_course_instance_meetings(instanceID: course_id)
   end
+
+private
+  def check_env_vars
+    if ENV["POPULI_API_URL"] == FAKE_POPULI_URL
+      Rails.logger.warn("WARNING: POPULI_API_URL environment variable is not set. Using a fake url. This may cause issues with features that utilize the Populi API")
```

Comment:
Nice 👍 I'm assuming anyone using the gem (probably just us at Turing) is using it in a Rails app, but would this throw and error if ran outside of Rails? Is there a more generic Ruby logger?

Probably not worth spending a ton of energy on, just not sure if anyone outside of Turing is using this gem, since it's technically public/open-source

Reply:
So this file is within the Present app so that shouldn't be an issue. If we wanted to add some warning messages to the Populi Gem, then yes we couldn't use the Rails loggers. I think writing to STDERR would be the way to go in that case.

Reply:
🤦‍♂️ welp I'm a dummy - this whole time I was thinking this was in the populi gem repo. Carry on, and sorry, lol